### PR TITLE
Clean-up SA packages and cobra commands

### PR DESCRIPTION
**Pull request by joshrosso**
This cleans up the package and commands in standalone to prepare for
feature freeze.

### CODE REVIEW

**Comment by joshrosso** [comment]
dunno how tagger go.mod got involved, will fix that shortly.

File: cli/cmd/plugin/standalone-cluster/cmd/list.go [hunk 1]
```diff
@@ -1,22 +1,26 @@
 // Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package main
+package cmd
 
 import (
 	"fmt"
-
```
Comment (stmcginnis) [inline cli/cmd/plugin/standalone-cluster/cmd/list.go:8]: Blank line should stay to separate the groups of "standard lib", "third party lib", "local" imports.

File: cli/cmd/plugin/standalone-cluster/cmd/utils.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
+package cmd
+
+import (
+	"github.com/spf13/pflag"
```
Comment (stmcginnis) [inline cli/cmd/plugin/standalone-cluster/cmd/utils.go:4]: Need to move this down to a separate third party group below.

File: cli/cmd/plugin/standalone-cluster/main.go [hunk 3]
```diff
@@ -4,23 +4,14 @@
...
-	"runtime"
-	"strconv"
-
-	"github.com/spf13/pflag"
-
+	"github.com/vmware-tanzu/community-edition/cli/cmd/plugin/standalone-cluster/cmd"
 	cliv1alpha1 "github.com/vmware-tanzu/tanzu-framework/apis/cli/v1alpha1"
 	"github.com/vmware-tanzu/tanzu-framework/pkg/v1/cli/command/plugin"
 	"github.com/vmware-tanzu/tanzu-framework/pkg/v1/tkg/log"
+	"os"
```
Comment (stmcginnis) [inline cli/cmd/plugin/standalone-cluster/main.go:11]: ^^

File: cli/cmd/plugin/standalone-cluster/cmd/delete.go [hunk 4]
```diff
@@ -1,26 +1,30 @@
 // Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package main
+package cmd
 
 import (
 	"fmt"
-
```
Comment (stmcginnis) [inline cli/cmd/plugin/standalone-cluster/cmd/delete.go:8]: Keep

File: cli/cmd/plugin/standalone-cluster/cmd/utils.go [hunk 5]
```diff
@@ -0,0 +1,41 @@
+package cmd
```
Comment (stmcginnis) [inline cli/cmd/plugin/standalone-cluster/cmd/utils.go:1]: Missing copyright header too.

**Pull request merged**